src/screens/professorHome/components/monitorsList/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { TSubjectMonitor } from '../../../../service/requests/useGetSubject/types';
+import MonitorListItem from '../monitorListItem';
+import {
+  Container,
+  FallbackTypography,
+  MonitorsListHeaderTypography,
+} from './styles';
+
+type Props = {
+  monitors: TSubjectMonitor[];
+};
+
+const MonitorsList = ({ monitors }: Props) => {
+  return (
+    <>
+      {monitors.length ? (
+        <Container>
+          <MonitorsListHeaderTypography variant="subtitle1">
+            Seus monitores
+          </MonitorsListHeaderTypography>
+          {monitors.map((monitor) => (
+            <MonitorListItem monitor={monitor} />
+          ))}
+        </Container>
+      ) : (
+        <FallbackTypography>
+          Você ainda não possui monitores(as) nem solicitações de monitoria
+          nesta disciplina.
+        </FallbackTypography>
+      )}
+    </>
+  );
```

Comment:
Sugestão pra melhorar legibilidade:

```suggestion

  if (!monitors.length) {
    return (
      <FallbackTypography>
          Você ainda não possui monitores(as) nem solicitações de monitoria
          nesta disciplina.
        </FallbackTypography>
    );
  }

  return (
    <Container>
      <MonitorsListHeaderTypography variant="subtitle1">
        Seus monitores
      </MonitorsListHeaderTypography>
      {monitors.map((monitor) => (
        <MonitorListItem monitor={monitor} />
      ))}
    </Container>
  );
```

Reply:
resolvido